indico/modules/rb/models/reservations.py

```diff
@@ -155,6 +159,11 @@ def __table_args__(cls):
         nullable=False,
         default=0
     )  # 1, 2, 3, etc.
+    recurrence_weekdays = db.Column(
+        ARRAY(db.String),
```

Comment:
I'm wondering if this could be an enum?

Reply:
As discussed: this would probably be a better candidate for a future PR/improvement task (one of the current downsides being that this PR is already set-up to just accept the weekday shorthands in lots of different places, whereas the upside being that we can remove the constraint for recurrence_weekdays).